storage/database/doc.go

```diff
@@ -14,5 +14,28 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package database implements various types of databased used in Klaytn.
+/*
+Package database implements various types of databases used in Klaytn.
+This package is used to read/write data from/to the persistent layer.
+
+DBManager is the interface used by the consumers of database package.
+databaseManager is the implementation of DBManager interface. It contains cacheManager and a list of Database interfaces.
+cacheManager caches data stored in the persistent layer, to decrease the direct access to the persistent layer.
+Database is the interface for persistent layer implementation. Currently there are 4 implementations, levelDB, memDB,
+badgerDB and partitionedDB.
+
+Source Files
+
+  - badger_database.go       : implementation of badgerDB, which wraps github.com/dgraph-io/badger
+  - cache_manager.go         : implementation of cacheManager, which manages cache layer over persistent layer
+  - db_manager.go            : contains DBManager and databaseManager
+  - interface.go             : interfaces used outside database package
+  - leveldb_database.go      : implementation of levelDB, which wraps github.com/syndtr/goleveldb
+  - memory_database.go       : implementation of MemDB, which wraps go native map structure
+  - metrics.go               : metrics used in database package, mostly related to cacheManager
+  - partitioned_database.go  : implementation of partitionedDB, which wraps a list of Database interface
+  - schema.go                :
```

Comment:
Does not `schema.go` need any description?

Reply:
My bad =( I will update this.